app/scripts/filters/resources.js

```diff
@@ -1029,6 +1029,22 @@ angular.module('openshiftConsole')
         return 'Terminating';
       }
 
+      // If pod.status.initContainerStatuses include state.terminated.reason with
+      // a value of "Error", display it instead of pod.status.containerStatus.
+      var initReason;
+
+      angular.forEach(pod.status.initContainerStatuses, function(initContainerStatus) {
+        var initContainerReason = _.get(initContainerStatus, 'state.terminated.reason');
```

Comment:
It looks like there is a `state.terminated.reason` even for successful init containters. We probably need to check `exitCode`, too.

```yaml
  initContainerStatuses:
    - containerID: >-
        docker://695625642cd6a776c8fe86a0ee530e0acbe04cfbb7187552c55b068f60df1333
      image: 'docker.io/openshift/origin-sti-builder:v3.9.0-alpha.3'
      imageID: >-
        docker-pullable://docker.io/openshift/origin-sti-builder@sha256:1120189696907b0451dcd081b39406be02c14b988cc7fc655af39002a0953ced
      lastState: {}
      name: git-clone
      ready: true
      restartCount: 0
      state:
        terminated:
          containerID: >-
            docker://695625642cd6a776c8fe86a0ee530e0acbe04cfbb7187552c55b068f60df1333
          exitCode: 0
          finishedAt: '2018-01-19T23:18:35Z'
          reason: Completed
          startedAt: '2018-01-19T23:18:33Z'
```
If there are multiple init containers and only the first init container has it, it looks like `initContainerReason` will be unset since we don't break the loop early.